Dockerfile

```diff
@@ -1 +1,14 @@
-FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/hello-world:latest
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base:20210701-2f4a4c7
+
+RUN  yum -y install java-11-openjdk
+
+ENV JAVA_HOME /usr/lib/jvm/java-11-openjdk-11.0.11.0.9-1.el7_9.x86_64
```

Comment:
Next time CentOS updates the Java-11-openjdk package, the JAVA_HOME will have a different version.  It could be like `/usr/lib/jvm/java-11-openjdk-11.0.1.0.42-1.el7_9.x86_64` (revision 42, instead of revision 9).  Something like that.

I think there should be a symlink we could use instead, something like `/usr/lib/jvm/java-11-openjdk`, which would always be a link to the installed version.

Reply:
updated with symlink